mp/src/game/client/momentum/ui/HUD/hud_speedometer.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "hud_speedometer_label.h"
+
+#include "hudelement.h"
+
+#include <vgui_controls/EditablePanel.h>
+#include "mom_player_shared.h"
+
+#define SPEEDOMETER_MAX_LABELS 4
+
+namespace vgui 
+{
+    class Label;
+}
+
+typedef CUtlVector <SpeedometerLabel *> SpeedoLabelList;
```

Comment:
You can keep the type brackets close to the class, this makes it look like there's three things here:

`CUtlVector<SpeedometerLabel*>`